folium/folium.py

```diff
@@ -235,6 +236,7 @@ def __init__(
         height: Union[str, float] = "100%",
         left: Union[str, float] = "0%",
         top: Union[str, float] = "0%",
+        font_size: str = "1rem",
```

Comment:
This needs to be on the bottom as the last argument so it does not break potential use of positional args.

It also needs to be added to the docstring above.

Reply:
Good catch, I've implemented these changes